nixos/maintainers/scripts/gce/create-gce.sh

```diff
@@ -7,9 +7,9 @@ BUCKET_NAME="${BUCKET_NAME:-nixos-cloud-images}"
 TIMESTAMP="$(date +%Y%m%d%H%M)"
 export TIMESTAMP
 
-nix-build '<nixpkgs/nixos/lib/eval-config.nix>' \
+nix-build '<nixpkgs/nixos>' \
    -A config.system.build.googleComputeImage \
-   --arg modules "[ <nixpkgs/nixos/modules/virtualisation/google-compute-image.nix> ]" \
+   --arg configuration "<nixpkgs/nixos/modules/virtualisation/google-compute-image.nix>" \
```

Comment:
I understand `<nixpkgs/nixos>` (i.e. `nixpkgs/nixos/default.nix`) has the sole purpose of making `<>`-style invocations ux-friendly. Still, the now official entrypoint is `nixos/lib`.

Reply:
Flakes shouldn't have stopped us from providing a better entrypoint that's pure and clean (wow, that sounds weird) and supports the old-cli `--arg` options. We might consider `__functor` on `<nixpkgs/nixos/lib>` for this, but I'm really not sure...